src/frontend/packages/markdown-components/package.json

```diff
@@ -21,14 +23,17 @@
   },
   "peerDependencies": {
     "@mui/material": "^5.13.1",
-    "@mui/system": "^5.13.1",
     "react": "^18.2.0",
     "react-admin": "^4.11.0"
   },
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
-    "parcel": "^2.9.3"
+    "@mui/material": "^5.14.11",
+    "@parcel/packager-ts": "^2.9.3",
+    "@parcel/transformer-typescript-types": "^2.9.3",
+    "parcel": "^2.9.3",
+    "react-admin": "^4.14.3"
```

Comment:
Why do you need to put `@mui/material` and `react-admin` in devDependencies when they are already on the peerDependencies ? Is it because of typescript ? It should at least be the same version between the two.

Reply:
Yes, as we import these dependencies in the component, we need at least their typings installed here in Semapps. It's the only solution I found to make it work, but I'm curious if you have a better one.

I fixed the versions mismatch!

Reply:
Any idea @Laurin-W ?

Reply:
Cool to see it's working in tsx :)

I did some research but I'm not aware of a different solution. This regards javascript as well, though. Neither for my IDE (vscode) nor for eslint, I get any code suggestions, if the packages are only installed as peer dependencies.

Two minor things I noted:
- I'd add react as a devDependency as well here.
- There are a couple of things the linter warns about that you could address or partially add ignore comments. I use the eslint vscode plugin to see them inline but you can see the issues for a file running `yarn run lint-files <path-to-file>` as well :)

Reply:
Thanks for your feedback. I fixed some eslint rules, and ignored those less relevant with Typescript.